### DISCUSSION TITLE
Opening question for meeting notes practice

### DISCUSSION THREAD

**Discussion by rebecca-vanhoeck** [SAEL-SWFSC/PAM-SI-Mtg-March2024#60] [Q&A]
Hi @shannonrankin and @Kourtney-Burger! What do you think about these options for an opening question to get folks practicing collaborating in a google doc? I tried to come up with at least a fun and productive option, but am open to other ideas too. 

Options: 
1. If you could have dinner with any scientist past or present who would it be and why?
2. What fun activities do you have planned or have you done in La Jolla/San Diego this week?
3. What are you most looking forward to about the PAM SI Workshop this week?

I've also drafted a few slides to introduce how we will be using the workshop notes [here ](https://docs.google.com/presentation/d/1oyJDp7lbEnDf0gHMjGSOvSwnOzazaXzIsJKrDTmezuM/edit?usp=sharing)(slides 12 - 15)

**Comment by Kourtney-Burger** [comment]
Hi Becca, I like the first question and I think the slides look great!

**Comment by shannonrankin** [comment]
Yes, agreed.
Also- will try to get into pull request for github changes-- but it's been
one thing or another.
Cheers,
Shannon
